internal/commands/app/delete.go

```diff
@@ -0,0 +1,80 @@
+package app
+
+import (
+	"github.com/10gen/realm-cli/internal/cli"
+	"github.com/10gen/realm-cli/internal/cloud/realm"
+	"github.com/10gen/realm-cli/internal/terminal"
+
+	"github.com/spf13/pflag"
+)
+
+// CommandDelete is the `app delete` command
+type CommandDelete struct {
+	inputs      cli.ProjectInputs
```

Comment:
We can delete multiple apps with the survey multi-select, but only accept one value for the --app flag. I _think_ this is what the spec implied, but I am open to different interpretation here

Reply:
I'm of the opinion if we support a procedure interactively, we should probably support it in the single command if possible. That being said I think the pattern should be `deleteInputs` with a `StringSlice`.

Reply:
see my comment about `ResolveApps` ... but it just doesn't seem like trying to use any of the `cli` helpers here is appropriate

Reply:
Done, refactored this part significantly